tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -0,0 +1,91 @@
+# SLE12 online migration tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "console_yasttest";
+use strict;
+use testapi;
+
+sub run {
+    my $self = shift;
+    select_console 'root-console';
+
+    script_run("/sbin/yast2 migration; echo yast2-migration-done-\$? > /dev/$serialdev", 0);
```

Comment:
Is this test only ever going to be run in text mode?

Don't we need to test it in the graphical environment also?